.github/pull_request_template.md

```diff
@@ -2,6 +2,7 @@ Please ensure your pull request follows these guidelines:
 
 - [ ] Link to the related issue with requirements and/or clearly describe 1) what problem this solves and 2) the requirements to review it against.
 - [ ] Update the `CHANGELOG.md` file with the added feature or fix. If there is not yet a heading for an upcoming release, add one following the established pattern.
-- [ ] Keep the pull request minimal! Break large updates into separate pull requests whenever possible. Small requests get reviewed more quickly.
+- [ ] Keep the pull request minimal! Break large updates into separate pull requests for individual features/fixes whenever possible. Small requests get reviewed more quickly.
+- [ ] All tests must pass, including the linters. Run `npm run lint` to test code linting independently.
```

Comment:
If the module does not yet have a `lint` task please adhere to the formatting style in place.

Reply:
@boutell This is a template only for core, which has the test.